controllers/azuremanagedcontrolplane_controller_test.go

```diff
@@ -176,3 +183,80 @@ func TestAzureManagedControlPlaneReconcilePaused(t *testing.T) {
 	g.Expect(err).To(BeNil())
 	g.Expect(result.RequeueAfter).To(BeZero())
 }
+
+func TestAzureManagedControlPlaneReconcileNormal(t *testing.T) {
+	g := NewWithT(t)
+	ctx := context.Background()
+	cp := &infrav1.AzureManagedControlPlane{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "fake-azmp",
+			Namespace: "fake-ns",
+		},
+		Spec:   infrav1.AzureManagedControlPlaneSpec{},
+		Status: infrav1.AzureManagedControlPlaneStatus{},
+	}
+	scheme, err := newScheme()
+	g.Expect(err).ToNot(HaveOccurred())
+
+	client := fake.NewClientBuilder().WithScheme(scheme).WithObjects(cp).WithStatusSubresource(cp).Build()
+	amcpr := &AzureManagedControlPlaneReconciler{
+		Client:   client,
+		Recorder: record.NewFakeRecorder(42),
```

Comment:
Is there a reason behind this particular channel buffer size of 42? Or is this even necessary at all if the test doesn't read recorded events?

Reply:
This doesn't even seem necessary.  The arbitrary 42 was chosen for no particular reason.  I can remove this line as it's not necessary.

Reply:
I removed this as this doesn't do anything for the test. Resolving.